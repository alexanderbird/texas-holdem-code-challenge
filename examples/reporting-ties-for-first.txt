2C 4C 6C 8S TS
Lexine 3D AS
Laney 3C KS
Francois 3H AH
Evelin 3S JH

---

2 Lexine Highcard Ace (kicker 10 8 6 4)
2 Francois Highcard Ace (kicker 10 8 6 4)
3 Laney Highcard King
4 Evelin Highcard Jack
