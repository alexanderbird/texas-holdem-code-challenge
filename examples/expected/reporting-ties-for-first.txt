2 Lexine Highcard Ace
2 Francois Highcard Ace
3 Laney Highcard King
4 Evelin Highcard Jack
